packages/ripple/index.tsx

```diff
@@ -26,7 +26,8 @@ import {Subtract} from 'utility-types'; // eslint-disable-line no-unused-vars
 // @ts-ignore no mdc .d.ts file
 import {MDCRippleFoundation, MDCRippleAdapter, util} from '@material/ripple/dist/mdc.ripple';
 
-const MATCHES = util.getMatchesProperty(HTMLElement.prototype);
+const HTMLElementShim: any = typeof HTMLElement === 'undefined' ? () => {} : HTMLElement;
```

Comment:
Shouldn't the default be an object instead of a function?
```
const HTMLElementShim: any = typeof HTMLElement === 'undefined' ? {} : HTMLElement;
```

Reply:
Sure, can you explain why for me? I don’t necessarily understand what the difference is. I’ll make the change real quick. 👍

Reply:
Done.

Reply:
On the next line you reference HTMLElementShim.prototype. That means that the HTMLElementShim should be an object - not a function.

Reply:
Thank you!